module/platform/Webots/src/Webots.cpp

```diff
@@ -564,6 +585,29 @@ namespace module::platform {
                             actuator_requests.motor_pids.emplace_back(
                                 MotorPID(servo.name, {servo.p_gain, servo.i_gain, servo.d_gain}));
                         }
+
+                        // Set the terminate command if the flag is set to terminate the simulator, used by the walk
+                        // simulator
+                        if (terminate_simulation) {
+                            std::cout << "setting terminate on ActuatorRequests to send" << std::endl;
```

Comment:
```suggestion
                            log<NUClear::DEBUG>("Setting terminate on ActuatorRequests to send.")
```
I'd also maybe suggest changing the text, it reads odd.